pyccel/ast/core.py

```diff
@@ -3277,6 +3282,7 @@ def rename(self, newname):
         self._name = newname
 
     def vectorize(self, body , header):
```

Comment:
we need to keep this in order to vectorize a function

Reply:
@saidctb Is the vectorizing currently implemented? It doesn't look like this function does the vectorizing. Are we sure that this is how the vectorizing will be done when it is implemented?

Reply:
I am also perplexed by this method, which is doing operations that do not make much sense to me.
For example, the line
```python
self._kind = procedure
```
should read
```python
self._kind = 'procedure'
```
Further, I don't understand why `body` should be an argument to this method, it looks like the vectorization is really performed by another function that we do not have. And finally, we do not even have a `vectorize` decorator!

@saidctb I am strongly in favor of removing this function... We can copy it to some "obsolete" folder if you really care about it.

Reply:
it's okay we can delete it